build.md

```diff
@@ -227,6 +227,28 @@ Cache target is defined using the same `type=TYPE[,KEY=VALUE]` syntax defined by
 
 Unsupported cache target MUST be ignored and not prevent user from building image.
 
+### additional_contexts
```

Comment:
In `docker-bake.hcl`, this is `contexts`. I wonder if we should try to remain consistent here (though having both `context` & `contexts` is 😬)

Reply:
In the linked issue me and @ndeloof discussed this. I was against keeping `contexts` because I think it's going to introduce confusion (context is mandatory, these are just additional contexts for the builder to use and is optional)

Reply:
(I'm okay with `named_contexts` as well)

Reply:
Further discussion in https://github.com/compose-spec/compose-spec/pull/307#discussion_r1110358705